docs/configuration/installation.rst

```diff
@@ -3,45 +3,9 @@
 Installation
 ############
 
-*Kinto* is meant to be relatively straightforward to install, and we've
-provided fairly extensive documentation to get you up and running. That said,
-some dependencies might make the process harder than it should be -
-we've got some notes on crypto libraries and PostgreSQL at the end of this page.
+Depending on the platform, and chosen configuration some libraries or
+extra services may become mandatory.
 
-Run locally
-===========
-
-To test Kinto quickly
----------------------
-
-In general, in order to run Kinto, it is as simple as running the following
-commands::
-
-    pip install kinto
-    wget https://raw.githubusercontent.com/Kinto/kinto/master/config/kinto.ini
-    pserve kinto.ini
```

Comment:
I still think installing through pip is the easiest for most users; we should probably keep promoting it first.